lib/trino-parquet/src/main/java/io/trino/parquet/ParquetEncoding.java

```diff
@@ -192,6 +196,22 @@ public boolean usesDictionary()
         {
             return true;
         }
+    },
+
+    BYTE_STREAM_SPLIT {
```

Comment:
We also need to update getDoubleDecoder and getRealDecoder in ValueDecoders to allow this encoding

Reply:
Done.